core/src/main/scala/org/bykn/bosatsu/Externals.scala

```diff
@@ -61,13 +67,15 @@ object FfiCall {
   def getJavaType(t: rankn.Type): List[Class[_]] = {
     def loop(t: rankn.Type, top: Boolean): List[Class[_]] = {
       t match {
-        case t if t == rankn.Type.IntType => classOf[java.lang.Integer] :: Nil
-        case t if t == rankn.Type.BoolType => classOf[java.lang.Boolean] :: Nil
+        //case t if t == rankn.Type.IntType => classOf[java.lang.Integer] :: Nil
+        //case t if t == rankn.Type.BoolType => classOf[java.lang.Boolean] :: Nil
```

Comment:
we can revisit the typing of the FFI later, but it looks like if we don't keep things as anyref, then things will be tricky to get right in generic contexts since I think we are currently resolving the final types. So, `foldLeft` in some contexts is using `Int` all the way through.